py/bitbox02/setup.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019 Shift Cryptosecurity AG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""BitBox python package"""
+from distutils.core import setup
+import setuptools
+
+with open("README.md", "r") as fh:
+    LONG_DESCRIPTION = fh.read()
+
+setup(
+    name="bitbox02",
+    version="0.0.1",
+    author="Shift Cryptosecurity",
+    author_email="support@shiftcrypto.ch",
+    packages=setuptools.find_packages(),
+    description="Python library for bitbox02 communication",
+    long_description=LONG_DESCRIPTION,
+    long_description_content_type="text/markdown",
+    url="https://github.com/digitalbitbox/bitbox02-firmware",
+    python_requires=">=3.6",
+    classifiers=[
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python :: 3.6",
+    ],
+    keywords="digitalbitbox bitbox bitbox02 bitcoin litecoin ethereum erc20 u2f",
+    install_requires=["hidapi", "noiseprotocol", "protobuf", "ecdsa", "semver"],
```

Comment:
this is missing version bounds like in requirements.py. I think the newest protobuf does not work yet with our code, for example.

Speaking of, is `requirements.py` still needed?

Reply:
Oh, I didn't know that you could put versions in the `setup.py` nowadays.

After reading `https://packaging.python.org/discussions/install-requires-vs-requirements/` it seems like we can remove the requirements files yes.